source/access/mfa_guide.rst

```diff
@@ -120,10 +120,16 @@ First of all, verify that your agent is running. You can do this by executing::
 
 If the agent is not running, you will get a
 ``Could not open a connection to your authentication agent.`` message. In this
-case you can start the agent with::
+case you can start an instance of the agent with::
 
     eval $(ssh-agent)
 
+.. note::
+   If you start your agent in this way, it is only accessible within the context of your 
+   current shell. If you want to connect with NoMachine NX, you should also start your 
+   NoMachine client from within this shell. Otherwise it will not be able to access the MFA
+   certificate stored in your agent.
```

Comment:
The rest of the page just uses the term 'certificate', not 'MFA certificate'. So to avoid potential confusion I think it'd be best to also just write 'certificate' here and in the other changes in this PR.